src/BSON/Timestamp.c

```diff
@@ -214,6 +215,43 @@ PHP_METHOD(Timestamp, __toString)
 }
 /* }}} */
 
+/* {{{ proto array Timestamp::jsonSerialize()
+*/
+PHP_METHOD(Timestamp, jsonSerialize)
+{
+	php_phongo_timestamp_t *intern;
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		return;
+	}
+
+	intern = Z_TIMESTAMP_OBJ_P(getThis());
+
+	array_init_size(return_value, 1);
+
+#if PHP_VERSION_ID >= 70000
+	{
+		zval ts;
+
+		array_init_size(&ts, 2);
+		ADD_ASSOC_LONG_EX(&ts, "t", intern->timestamp);
+		ADD_ASSOC_LONG_EX(&ts, "i", intern->increment);
```

Comment:
I think HHVM does the same format too:

```
        return [
            '$timestamp' => [
                't' => (int) $this->timestamp,
                'i' => (int) $this->increment,
            ]
        ];
```

So nothing we should change there - right?